service/src/main/java/bio/terra/pipelines/db/JobsDao.java

```diff
@@ -133,7 +136,7 @@ public List<Job> getJobs(String userId, String pipelineId) {
   private Optional<DbJob> getDbJobIfExists(String userId, String pipelineId, String jobId) {
     final String sql =
         """
-                        SELECT job_id, user_id, pipeline_id, pipeline_version, time_submitted, time_completed, status
+                        SELECT job_id, user_id, pipeline_id, pipeline_version, time_submitted, time_completed, status, pipeline_inputs
```

Comment:
Was thinking we might want to separate the pipeline_inputs into its own table so we don't have to keep returning that data (which may be large in some cases) when we just want to return the job information like we currently do for the 2 jobs GET endpoints.  Something simple like

|job_id|pipeline_inputs|
|---|---|
|UUID|Text|